salt/modules/opkg.py

```diff
@@ -412,12 +453,19 @@ def install(name=None,
             ret.update({pkgname: {'old': old.get(pkgname, ''),
                                   'new': new.get(pkgname, '')}})
 
+    rs_result = __salt__['restartcheck.restartcheck'](verbose=False)
+    if isinstance(rs_result, dict) and 'comment' in rs_result:
```

Comment:
We have this duplicated over on L527. Could we refactor it to a single block that's called?

Reply:
Done.